.gitlab/integration_test/windows.yml

```diff
@@ -26,7 +26,6 @@
       -e CI_PIPELINE_ID="${CI_PIPELINE_ID}"
       -e CI_PROJECT_NAME="${CI_PROJECT_NAME}"
       -e AWS_NETWORKING=true
-      -e PY_RUNTIMES="$PYTHON_RUNTIMES"
```

Comment:
As a consequence, I suppose we can also clean occurrences of `PY_RUNTIMES`. I was thinking of the `winbuildscripts` and [it seems](https://github.com/search?q=repo%3ADataDog%2Fdatadog-agent%20PY_RUNTIMES&type=code) there is also some possible cleanup in omnibus.

Reply:
This is handled by another commit in the original PR: https://github.com/DataDog/datadog-agent/pull/30035/commits/2b76319b42afce04e21c731b03c5a9ebe3f33b77